### PR TITLE
refresh page after website creation

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-FEATURE] refresh the page after website creation to show it in the fiexed top nav list

### CODE REVIEW

**Comment by alavieille** [comment]
you can reload in the submit admin, this file is generic, it's open close.

**Comment by djamnazi** [comment]
@alavieille  PR updated

File: BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+###*
+ * @class WebSiteFormView
+###
+class OpenOrchestra.WebSite.WebSiteFormView extends FullPageFormView
+
+  ###*
+   * On Element Created
+  ###
+  onElementCreated: ->
+    document.location.reload 1 if @options.entityType == 'websites'
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee]: the entity is always websites in this case

File: GruntTasks/Targets/concat.orchestra_js.js [hunk 2]
```diff
@@ -121,6 +121,10 @@ module.exports = {
     'web/built/openorchestrabackoffice/js/user/UserFormView.js',
     'web/built/openorchestrauseradmin/js/user/*.js',
 
+    //--[ WEBSITE ]--//
+    'web/built/openorchestrabackoffice/js/webSite/WebSiteFormView.js',
+    'web/built/openorchestrauseradmin/js/webSite/*.js',
```
Comment (alavieille) [inline GruntTasks/Targets/concat.orchestra_js.js]: remove this line

File: BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee (hunk 1 above)
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee]: why 1 and not a boolean ?

Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee]: I agree with these two comments

**Comment by djamnazi** [comment]
@alavieille PR updated

File: BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+###*
+ * @class WebSiteFormView
+###
+class OpenOrchestra.WebSite.WebSiteFormView extends FullPageFormView
+
+  ###*
+   * On Element Created
+  ###
+  onElementCreated: ->
+    document.location.reload 1
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/webSite/WebSiteFormView.coffee]: why not  document.location.reload true ?

**Comment by alavieille** [comment]
:+1:

**Pull request merged**